app/models/user.rb

```diff
@@ -117,8 +117,8 @@ def self.create_with_identity(auth, token)
 
   def self.find_from(auth)
     user = User.joins(:identities)
-           .merge(Identity.where(provider: auth[:provider], uid: auth[:uid]))
-           .first
+               .merge(Identity.where(provider: auth[:provider], uid: auth[:uid]))
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.